code/obj/landmark.dm

```diff
@@ -642,6 +642,10 @@ var/global/list/job_start_locations = list()
 	warptarget_modifier = LANDMARK_VM_WARP_NONE
 /// target turf for projecting its contents elsewhere
 /turf/var/turf/vistarget = null
+/// turfs to project speech to (overlaps with vistarget speech)
+/turf/var/list/listening_turfs = null
```

Comment:
When you say it overlaps, why is this necessary instead of just relying on the existing vistarget behaviour?

Reply:
I didn't want to override vistarget in case anything else was using it (I thought it made sense for things like Centcomm, but on station might be different)